junit-platform-engine/src/main/java/org/junit/platform/engine/support/store/NamespacedHierarchicalStore.java

```diff
@@ -245,18 +246,21 @@ public <K, V> Object computeIfAbsent(N namespace, K key, Function<? super K, ? e
 		var result = StoredValue.evaluateIfNotNull(storedValue);
 		if (result == null) {
 			StoredValue newStoredValue = this.storedValues.compute(compositeKey, (__, oldStoredValue) -> {
```

Comment:
```suggestion
			StoredValue newStoredValue = storedValues.compute(compositeKey, (__, oldStoredValue) -> {
```
the best we can do is to do nothing at all, or do whats close to that meaning delete.

everything we can delete without breaking seems obsolet, mostly imposing unwanted coupling burden.